### PR TITLE
Updated load_1D.py

### PR DESCRIPTION
**Pull request by JakHolzer**
Try, except deleted.
Added one if/elif loop to tell apart zebra mode (bi, nb) and different angles saved

### CODE REVIEW

**Comment by ivan-usov** [comment]
@JakHolzer , how about to use `is_integer()` (https://docs.python.org/3/library/stdtypes.html#float.is_integer) instead? I don't think that complicated logic with `Decimal` is needed.

File: pyzebra/load_1D.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                        time.append(float(data[position + 3].split()[6]))
+                    det_variables["Measurements"]["omega"] = omega
+                    det_variables["Measurements"]["counts"] = counts
+                    det_variables["Measurements"]["Monitor1"] = monitor1
+                    det_variables["Measurements"]["Monitor2"] = monitor2
+                    det_variables["Measurements"]["Monitor3"] = monitor3
+                    det_variables["Measurements"]["time"] = time
+                else:
+                    print("Unknown file extention")
+    if all(decimal):
```
Comment (ivan-usov) [inline pyzebra/load_1D.py:165]: Reading 'dat' files breaks at this line. Indentation is missing.

**Review by ivan-usov — CHANGES_REQUESTED** [review]
I left a few comments, but not sure they didn't result in a review.

**Review by ivan-usov — APPROVED** [review]

**Comment by JakHolzer** [comment]
The reason is that the hkl are in ccl in float format even for hkl, they
are for example -1.000 1.000 0.000, therefore they will be evaluated as
floats, not integer if I'm not mistaken. The decimal checks if there are
some numbers except zeros after decimal points which is only way how to
separate hkl from real indices.


pá 11. 9. 2020 v 14:07 odesílatel Ivan Usov <notifications@github.com>
napsal:

**Pull request merged**